lib/cogctl/actions/relay_groups/create.ex

```diff
@@ -40,7 +40,7 @@ defmodule Cogctl.Actions.RelayGroups.Create do
 
   defp add_to_group(group, relays, endpoint) do
     Enum.flat_map_reduce(relays, 0, fn(relay, acc) ->
-       case CogApi.HTTP.Client.relay_group_add_relay(%{name: group.name}, %{relay: relay}, endpoint) do
+       case CogApi.HTTP.Client.relay_group_add_relays(%{name: group.name}, %{relays: [relay]}, endpoint) do
```

Comment:
So it looks like this was done in order to get a slightly different behavior. Based on the ticket that this work was originally part of, https://github.com/operable/cog/issues/535. We want to add relays to the group that we can and return an error to the user for those that we can't. If you send all relays at once it's an all or none operation. I think that there is probably some room for improvement here, but I'm thinking that we should save it until we can standardize on what our output in situations should be like. Thoughts?